src/mumble/ConnectDialog.h

```diff
@@ -150,7 +150,7 @@ class ServerItem : public QTreeWidgetItem, public PingStats {
 	ServerItem(const PublicInfo &pi);
 	ServerItem(const QString &name, const QString &host, unsigned short port, const QString &uname,
 			   const QString &password = QString());
-#ifdef USE_BONJOUR
+#ifdef USE_ZEROCONF
 	ServerItem(const BonjourRecord &br);
```

Comment:
`BonjourRecord` (naming)?

Reply:
The structure is defined in `qqbonjour`.